src/applications/ezr/components/ConfirmationPage/ConfirmationScreenView.jsx

```diff
@@ -13,8 +12,6 @@ const ConfirmationScreenView = ({ name, timestamp }) => {
     scrollToTop();
```

Comment:
Replace this useEffect with `useEffect(() => focusElement('h2', {}, 'va-alert'), []);`. `scrollToTop` isn't necessary when focus is being set and it'll be better to apply focus to the alert heading.